mlx_graphs/datasets/lanl_dataset.py

```diff
@@ -0,0 +1,320 @@
+import os
+import shutil
+from typing import Optional
+
+import mlx.core as mx
+import numpy as np
+
+try:
+    import pandas as pd
+except ImportError as e:
+    raise ImportError(
+        (
+            "pandas is required to use this feature. "
+            "Install it with `pip install pandas pyarrow`"
+        )
+    ) from e
```

Comment:
This will cause problems when compiling the documentation. The two options are:
- import them inside the methods where they're needed (preferred IMO)
- add them to the default dependencies

Reply:
@TristanBilot on this, to make it work with the docs, we just need to mock pandas in `docs/source/conf.py` (as I did for networkx).

Reply:
done @francescofarina.